vm/systemSmartContracts/auction.go

```diff
@@ -196,6 +208,7 @@ func (s *stakingAuctionSC) changeRewardAddress(args *vmcommon.ContractCallInput)
 	isError := err != nil || vmOutput.ReturnCode != vmcommon.Ok
 	if isError {
 		log.LogIfError(err)
+		s.eei.SetReturnMessage(vmOutput.ReturnMessage)
```

Comment:
is vmoutput is nil this will result in panic. add err.error if err != nil. and vmoutput.Return message if vmoutput is not nil.

Reply:
refactored this part of code